Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Build driver
-FROM golang:1.10.1-alpine3.7 as builder
+FROM golang:1.13.4-alpine3.10 as builder
```

Comment:
How does this square with the 1.14 in the go.mod below?

Do we even need the go.mod if everything is in vendor/? I'm still a little vague, maybe there are other deps in go.mod (?)

Reply:
To use "go mod vendor" command, we need go.mod file to be available.

I checked with @msau42 , The plan is to checkin go.mod, go.sum and the vendor folder.

Reply:
Using 1.14.4 as base image for golang now